docs/docsite/rst/reference_appendices/glossary.rst

```diff
@@ -270,16 +275,16 @@ when a term comes up on the mailing list.
         servers to one particular server.
 
     Local Action
-        A local_action directive in a :term:`playbook <playbooks>` targeting
-        remote machines means that the given step will actually occur on the
-        local machine, but that the variable ``{{ ansible_hostname }}`` can be
-        passed in to reference the remote hostname being referred to in that
-        step.  This can be used to trigger, for example, an rsync operation.
+        The local_action keyword is an alias for ``delegate_to: localhost``.
+        Used when you want to redirect an action from the remote machines to
+        execute on the controller itself.
 
     Local Connection
         By using ``connection: local`` in a :term:`playbook <playbooks>`, or
         passing ``-c local`` to :command:`/usr/bin/ansible`, this indicates
-        that we are managing the local host and not a remote machine.
+        that we are executing a local fork instaed of executing on the remote machine.
```

Comment:
"instaed" -> "instead".
instead of "remote machine" should we use "remote target" ?